docs/1-GettingStarted.md

```diff
@@ -60,25 +60,32 @@ By adding these permissions [Google Play will automatically filter out devices](
 ## Architecture
 
 ### What's with this .Current Global Variable? Why can't I use $FAVORITE_IOC_LIBARY
-You totally can! Every plugin I create is based on an interface. The static singleton just gives you a super simple way of gaining access to the platform implementation. Realize that the implementation of the plugin lives in your iOS, Android, Windows, etc. Thies means you will need to register it there by instantiating a `CrossConnectivityImplementation` from the platform specific projects.
+You totally can! Every plugin I create is based on an interface. The static singleton just gives you a super simple way of gaining access to the platform implementation for quick throw-away proof of concept applications. When you are using dependency inversion (i.e the _D_ in [SOLID](https://en.wikipedia.org/wiki/SOLID_(object-oriented_design))) software architecture then your viwmodels, services and any code that needs access to the functionality provided by this plugin must be via the `IConnectivity` interface which is passed in as an argument in your constructor and assigned to a readonly private member to prevent accidential value reassignment.
 
-If you are using a ViewModel/IOC approach your code may look like:
 
 ```csharp
 public MyViewModel()
 {
-    readonly IConnectivity connectivity;
+    readonly IConnectivity _connectivity;
     public MyViewModel(IConnectivity connectivity)
     {
-        this.connectivity = connectivity;
+        _connectivity = connectivity;
     }
 
-    public bool IsConnected => connectivity?.IsConnected ?? false;
+    public bool IsConnected => _connectivity.IsConnected;
 }
```

Comment:
It's important that we don't hide failures. The viewmodel should go boom if instance isn't supplied into the constructor.